Quadriga/src/main/java/edu/asu/spring/quadriga/domain/impl/ConceptpowerReply.java

```diff
@@ -96,7 +99,29 @@
     public void setConceptEntry(List<ConceptpowerReply.ConceptEntry> entries) {
         this.conceptEntry = entries;
     }
+    
+    @XmlElement(required = true, namespace="http://www.digitalhps.org/")
+    protected List<ConceptpowerReply.AlternativeIds> alternativeIds;
```

Comment:
That doesn't make sense. AlternativeIds is a property of ConceptEntry. Then the class structure is wrong. Otherwise you would endup with a list of concept entries and a list of alternative ids, but you want each entry to have a list of alternative ids. You probably don't even need its own class. Just parse the elements inside alternativeIds into a list of strings (holding the ids).